### ISSUE TITLE
question regarding EGCL layer

### ISSUE THREAD

**Issue by orgw** [ehoogeboom/e3_diffusion_for_molecules#7]
why is normalizing factor 100 in code instead of distance?? in equation 12

**Comment by orgw** [comment]
plus in node_model you seem to do this:

agg = torch.cat([x, agg, node_attr], dim=1)
